starters/next13-zustand-bulma/src/components/Greeting/Greeting.tsx

```diff
@@ -0,0 +1,34 @@
+async function getMessage(greeting: string | undefined) {
+  const response = await fetch(
+    `https://api.starter.dev/.netlify/functions/server/hello?greeting=${greeting}`
+  );
+
+  if (!response.ok) {
+    const bodyText = await response.text();
+    const bodyJson = bodyText ? JSON.parse(bodyText) : null;
```

Comment:
Do we get a JSON error here?
Do we use `await response.text();` instead of `await response.json();` because it may not be a json error?

Reply:
I copy this part from the next 12 kit, so I assumed that the error may not be a json!